entrypoint.sh

```diff
@@ -29,7 +30,9 @@ apt -y install \
   git \
   cppcheck \
   python3-pip \
-  $APT_DEPENDENCIES
+  $OLD_APT_DEPENDENCIES \
+  $(sort -u $APT_DEPENDENCIES | tr '\n' ' ')
```

Comment:
I think that this should also be run against all of the dependencies from source, to install transitive dependencies without having to specify them directly in each package.

Reply:
Oh gotcha, so that's why your implementation goes over all the `packages.apt`. Let me do the same.

Reply:
Precisely, it's like a very simple variant of `rosdep install --from-path`